cluster/manifests/z-karpenter/configmap.yaml

```diff
@@ -0,0 +1,64 @@
+{{ if eq .Cluster.ConfigItems.karpenter_pools_enabled "true"}}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: karpenter-global-settings
+  namespace: kube-system
+  labels:
+    application: kubernetes
+    component: karpenter
+data:
+  "aws.clusterEndpoint": "https://kubernetes.default.svc.cluster.local."
+  "aws.clusterName": "{{ .Cluster.ID  }}"
+  "aws.defaultInstanceProfile": "{{ .Cluster.ID | awsValidID }}-WorkerKarpenter-InstanceProfile"
+  "aws.enableENILimitedPodDensity": "true" # DEPRECATED: Use `.spec.kubeletConfiguration.maxPods` to set pod density on a per-provisioner basis
+  "aws.enablePodENI": "false"
+  "aws.interruptionQueueName": "{{.Cluster.LocalID}}-karpenter-interruption-queue"
+  "aws.isolatedVPC": "false"
+  "aws.nodeNameConvention": "ip-name"
+  "aws.vmMemoryOverheadPercent": "0.075"
+  #The global tags to use on all AWS infrastructure resources (launch templates, instances, etc.) across node templates
+  "aws.tags": '{"InfrastructureComponent":"true","application":"kubernetes","component":"worker","environment":"{{ .Cluster.Environment }}"}'
```

Comment:
@mikkeloscar 
the alternative way to set the tags on all AWS infrastructure resources.
I gathered all the static/generic tags here and left the node-pool specific ones in the AWSNodeTemplate

Reply:
Great!